tools/docker/docker-compose-stage.yaml

```diff
@@ -103,8 +103,6 @@ services:
       postgres:
         condition: service_healthy
     restart: always
-    volumes:
-      - ${EDA_HOST_PODMAN_SOCKET_URL:-/run/user/1000/podman/podman.sock}:/run/podman/podman.sock:z
```

Comment:
Why are we removing volumes here but keeping it for `eda-default-worker`?

Reply:
It should be removed in both. good catch. Updated.